docs/examples/list-filtering/sqlalchemy/sqlalchemy_example/test_authorization.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+
+import pytest
+
+from sqlalchemy import create_engine
+
+from oso import Oso
+from sqlalchemy_oso.hooks import authorized_sessionmaker
+from sqlalchemy_oso.auth import register_models
+
+from .models import Post, User, Model
+
+POLICY_FILE = Path(__file__).absolute().parent / 'policy.polar'
+
+@pytest.fixture
+def engine():
+    engine = create_engine('sqlite:///:memory:')
+    Model.metadata.create_all(engine)
+    return engine
+
+@pytest.fixture
+def authorization_data():
+    return {
+        'user': None,
+        'action': "read"
+    }
+
+@pytest.fixture
+def session(engine, oso, authorization_data):
+    return authorized_sessionmaker(
+        bind=engine,
+        get_oso=lambda: oso,
+        get_user=lambda: authorization_data['user'],
+        get_action=lambda: authorization_data['action']
+    )()
+
+@pytest.fixture
+def oso():
+    return Oso()
+
+@pytest.fixture
+def policy(oso):
+    register_models(oso, Model)
+    oso.load_file(POLICY_FILE)
+
+@pytest.fixture
+def test_data(session):
+    user = User(username='user')
+    manager = User(username='manager', manages=[user])
+
+    public_user_post = Post(contents='public_user_post',
+                            access_level='public',
+                            created_by=user)
+    private_user_post = Post(contents='private_user_post',
```

Comment:
```suggestion
    public_user_post = Post(contents='public user post',
                            access_level='public',
                            created_by=user)
    private_user_post = Post(contents='private user post',
```

Reply:
This is going to change docs, tests, etc. I know it's inconsistent unfortunately.  Writing down as follow up.